extensions/smallrye-stork/runtime/src/main/java/io/quarkus/stork/StorkConfigUtil.java

```diff
@@ -1,14 +1,35 @@
 package io.quarkus.stork;
```

Comment:
Same here.

And given the class was preexisting, we probably need to take a step back and maybe change it in a separate PR after this is in.